src/renderer/components/InstanceModal/ConfirmDelete.ts

```diff
@@ -1,24 +1,49 @@
+import { LitElement, customElement, property, html } from "lit-element";
 import Instance from "../../Instance";
-import instanceDeleteModalTemplate from "../../templates/modals/instances/ConfirmDelete.pug";
 
+import assert from "assert";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['assert' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5396733)